.travis.yml

```diff
@@ -11,17 +11,12 @@ before_install:
 install:
   - npm ci
 
-before_script:
-  - npm run lint
-
 jobs:
   include:
-    - stage: eslint
+    - stage: lint
       script: npm run lint
     - stage: size
       script: npm run size
-    - stage: test
```

Comment:
No more testing?

Reply:
It's actually being done by default. 
This will then remove one of the duplicate node 8 test. 
See it here : The two duplicate node8 test https://travis-ci.com/dashevo/wallet-lib/builds/130726927

Reply:
ah right